turbinia/processors/mount_local.py

```diff
@@ -71,10 +71,8 @@ def GetDiskSize(source_path):
 
   if size is None:
     # evidence is not a block device, check image file size
-    cmd = ['ls', '-s', source_path]
     try:
-      cmd_output = subprocess.check_output(cmd).split()
-      size = int(cmd_output[0].decode('utf-8'))
+      size = os.stat(source_path).st_size
     except subprocess.CalledProcessError as exception:
```

Comment:
Since you removed the call to subprocess, the exception here would likely be something thrown by os.stat, possibly FileNotFoundError or other IO error.